operations/research-and-development/product/product-planning/launch-plan.md

```diff
@@ -1,7 +1,9 @@
 # Launch Plan
 
-We plan periodic launches to drive awareness in the market so that our target audiences who are not already Mattermost customers can get exposure to our product and our offerings. Launches are also a way for the company to put stories into the market to communicate Mattermost’s position and strategic direction. Launches are different than releases or a roadmap in that in a launch we highlight the most exciting changes to our product. A launch plan is owned by the marketing team whereas the release plan is owned by the release team (R&D). 
+We plan periodic launches to drive awareness in the market so that our target audiences who aren't already Mattermost customers can get exposure to our product and our offerings.
 
-We may plan a launch with the release of a new product offering, at the release of a highly demanded feature, or to announce a set of features that are thematically connected. The timing of a launch may be different than a release depending on other communications and events of the organization.  
+Launches are also a way for the company to put stories into the market to communicate Mattermost’s position and strategic direction. Launches are different than releases or a roadmap in that a launch highlights the most exciting changes to our product. A launch plan is owned by the Marketing team whereas the release plan is owned by the Release team (R&D).
+
+We may plan a launch with the release of a new product offering, at the release of a highly demanded feature, or to announce a set of features that are thematically connected. The timing of a launch may be different than a release depending on other communications and events of the organization.
 
 You can find information about [upcoming launches here](https://mattermost.atlassian.net/wiki/spaces/MKT/pages/521371683/Mattermost+Go-To-Market+GTM+Plan+-+FY21H1).
```

Comment:
This looks like an older page - do we have a newer one?

Reply:
@justinegeffen let's remove the reference and we can replace it if/when we start a new calendar specifically for launches.